### PR TITLE
feat: implement EsmeraldatorrentScraper

### PR DESCRIPTION
**Pull request by gecm0**
Añado soporte para [Esmeralda Torrent](https://esmeraldatorrent.com). Se trata de un sitio espejo a GranTorrent, por lo que el codigo es similar 👍🏻

### CODE REVIEW

File: src/scrapers/esmeraldatorrentScraper.ts [hunk 1]
```diff
@@ -0,0 +1,245 @@
...
+      for (const value of items) {
+        const foundTitle = slugify($(value).text());
+        if (foundTitle === slugify(title)) {
+          const magnet = await this.getMovieInfo($(value).attr('href') as string, year);
+          if (magnet) results.push(magnet);
+        }
+      }
+      return results;
+    } catch (error) {
+      console.log('DT - ERROR', error);
```
Comment (kekazoh) [inline src/scrapers/esmeraldatorrentScraper.ts:76]: [Sand] DT stands for DonTorrent, we could use ET or EsmeraldaTorrent for this one

File: src/scrapers/esmeraldatorrentScraper.ts [hunk 2]
```diff
@@ -0,0 +1,245 @@
...
+                };
+                return magnet;
+              }
+            }
+          }
+        }
+      }
+      return null;
+    } catch (error) {
+      console.log(`ERROR DONTORRENT getMovieInfo: ${error}`);
```
Comment (kekazoh) [inline src/scrapers/esmeraldatorrentScraper.ts:121]: [Sand] Same here

**Review by kekazoh — CHANGES_REQUESTED** [review]
I left a couple of small comments and a concern:
Does this site gives us different links to what dontorrent already do? If not I see no point on adding this new site, with duplicated code. If the site is a mirror version of dontorrent we could always change the `BASE_URL` and use the Dontorrent scraper, right?

**Comment by gecm0** [comment]
> I left a couple of small comments and a concern: Does this site gives us different links to what dontorrent already do? If not I see no point on adding this new site, with duplicated code. If the site is a mirror version of dontorrent we could always change the `BASE_URL` and use the Dontorrent scraper, right?

Agreed. We can just use the DonTorrentScraper class and change the `BASE_URL` 👍🏻 I proceed to close this PR

**Pull request closed without merging**